packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -120,6 +97,38 @@ contract Deploy is Deployer {
         }
     }
 
+    /// @notice Deploy all of the proxies
+    function deployProxies() public {
+        deployAddressManager();
+        deployProxyAdmin();
+
+        deployOptimismPortalProxy();
+        deployL2OutputOracleProxy();
+        deploySystemConfigProxy();
+        deployL1StandardBridgeProxy();
+        deployL1CrossDomainMessengerProxy();
+        deployOptimismMintableERC20FactoryProxy();
+        deployL1ERC721BridgeProxy();
+        deployDisputeGameFactoryProxy();
+
+        transferAddressManagerOwnership();
```

Comment:
ibid - preference to keep this outside the deployProxies function and call at the top-level.

Reply:
We need a single function that can be used to deploy the proxies when adding a chain to the superchain

Reply:
Ah ok, this intuition makes sense. Thanks for the clarification!